### PR TITLE
ERP CHP fuel HHV bug

### PR DESCRIPTION
**Pull request by hdunham**
### Fixed
 - When calculating CHP fuel intercept and slope, use 1 for the HHV because CHP fuel measured in units of kWh, instead of using non-existent **CHP.fuel_higher_heating_value_kwh_per_gal**
 ### Changed
 - Renamed function `generator_fuel_slope_and_intercept` to `fuel_slope_and_intercept` and generalize to not be specific to diesel measured in units of gal

### CODE REVIEW

File: src/constraints/outage_constraints.jl [hunk 1]
```diff
@@ -95,10 +95,10 @@ end
 
 
 function add_MG_Gen_fuel_burn_constraints(m,p)
-	fuel_slope_gal_per_kwhe, fuel_intercept_gal_per_hr = generator_fuel_slope_and_intercept(
+	fuel_slope_gal_per_kwhe, fuel_intercept_gal_per_hr = fuel_slope_and_intercept(
```
Comment (Bill-Becker) [inline src/constraints/outage_constraints.jl:133]: Should we do this for CHP fuel burn below as well?

File: CHANGELOG.md [hunk 2]
```diff
@@ -27,6 +27,9 @@ Classify the change according to the following categories:
 ### Fixed 
 - Added `export_rate_beyond_net_metering_limit` to list of inputs to be converted to type Real, to avoid MethodError if type is vector of Any. 
 - Fix blended CRB processing when one or more load types have zero annual energy
+- When calculating CHP fuel intercept and slope, use 1 for the HHV because CHP fuel measured in units of kWh, instead of using non-existent **CHP.fuel_higher_heating_value_kwh_per_gal**
```
Comment (Bill-Becker) [inline CHANGELOG.md:30]: Where was this implemented?

Comment (hdunham) [reply CHANGELOG.md:30]: https://github.com/NREL/REopt.jl/blob/54dbb3f226f873f0b8ba415a3d43be1f856e3608/src/outagesim/backup_reliability.jl#L803

File: src/constraints/outage_constraints.jl (hunk 1 above)
Comment (hdunham) [reply src/constraints/outage_constraints.jl:133]: Yeah sure! Added it in two more places to eliminate dupe code

**Review by Bill-Becker — APPROVED** [review]
Not sure why the Ubuntu tests always take forever since the switch to all tests with HiGHS, while the Windows and Mac ones finish reasonably quickly.

**Pull request merged**